dropwizard-app/src/main/java/org/coner/hibernate/entity/CompetitionGroup.java

```diff
@@ -0,0 +1,71 @@
+package org.coner.hibernate.entity;
+
+import org.hibernate.annotations.GenericGenerator;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.math.BigDecimal;
+
+/**
+ *
+ */
+@Entity
+@Table(name = "competition_groups")
+public class CompetitionGroup extends HibernateEntity {
+
+    private String id;
+    private String name;
+    private BigDecimal handicapFactor;
+    private boolean grouping;
+    private String resultTimeType;
+
+    @Id
+    @GeneratedValue(generator = "uuid")
+    @GenericGenerator(name = "uuid", strategy = "uuid2")
+    public String getId() {
+        return id;
+    }
+
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    @Column(name = "name")
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    @Column(name = "handicap_factor")
```

Comment:
It seems that database engines vary quite a bit in their support for case sensitive table and column names, so I think it would be good to use snake case for these.